docs/how-to-connect/plugin.md

```diff
@@ -12,5 +12,6 @@ Casdoor also provides plugins or middlewares for some very popular platforms, li
 | Spring Boot example | Java     | https://github.com/casdoor/casdoor-spring-boot-example |
 | WordPress plugin    | PHP      | https://github.com/casdoor/wordpress-casdoor-plugin    |
 | Odoo plugin         | Python   | https://github.com/casdoor/odoo-casdoor-oauth          |
+| Django plugin       | Python   | https://github.com/casdoor/django-casdoor-auth         |
 
-For a full list of the official Casdoor plugins, please see: https://github.com/casdoor?q=plugin&type=all&language=&sort=
\ No newline at end of file
+For a full list of the official Casdoor plugins, please see: [Casdoor repositories](https://github.com/casdoor?q=plugin&type=all&language=&sort=).
```

Comment:
https://github.com/casdoor?q=plugin&type=all&language=&sort=
It's not a full list of the official Casdoor plugins, maybe we need to remove it, or find a new solution

![image](https://user-images.githubusercontent.com/53544726/187448600-cff7b1d8-54f3-48f1-81b9-40cd2b88458c.png)

Reply:
I use sdk in:readme as query word, it can lists most of the plugins